generation/generate-test.go

```diff
@@ -209,6 +213,11 @@ func GenerateTest(args Args, expectations map[string][]string, outputWriter io.W
 	for _, pair := range hopsPairs {
 		g.batchConnect(pair)
 	}
+	// issue another round of connections to be sure we have flooded the network & receive all data from the hops
+	// (short-circuits a scheduling problem by paying with more execution time)
```

Comment:
maybe loose a sentence on what the problem is so that someone knows in a year?

> short-circuits figuring out which peer should replicate which one based on the follows hierarchy

something something..?

Reply:
good call ^^

Reply:
done https://github.com/ssb-ngi-pointer/netsim/pull/19/commits/2b2941f0a9487c035f42272910e2a54defd5fecb